Sources/Camille/main.swift

```diff
@@ -6,6 +6,20 @@ let bot = try SlackBot(
     authenticator: OAuthAuthentication.self,
     storage: RedisStorage.self,
     services: [
+        CrossPostService(config: CrossPostServiceConfig(
+            timeSpan: 60 * 2,
+            reportingTarget: "admins",
+            publicWarning: { channel, user in
+                return try SlackMessage()
+                    .line(user, " cross posting is discouraged.")
+                    .makeChatPostMessage(target: channel)
+            },
+            privateWarning: { im in
+                return try SlackMessage()
+                    .line("Please refrain from cross posting, it is discouraged here.")
+                    .makeChatPostMessage(target: im)
+            }
+        )),
```

Comment:
Will the Remove All option ever get included in the alert? I couldn't see how it ends up in there so I just wanted to check and make sure.